packages/blockchain-wallet-v4-frontend/src/modals/Staking/AccountSummary/AccountSummary.template.success.tsx

```diff
@@ -309,47 +309,55 @@ const AccountSummary: React.FC<Props> = (props) => {
-                            days:
-                              bondingDays > 1 ? (
-                                <FormattedMessage
-                                  defaultMessage='days'
-                                  id='modals.staking.warning.content.subtitle.days'
-                                />
-                              ) : (
-                                <FormattedMessage
-                                  defaultMessage='day'
-                                  id='modals.staking.warning.content.subtitle.day'
-                                />
-                              )
-                          }}
-                        />
-                      }
-                      text={
-                        <FormattedMessage
-                          defaultMessage='Stake {value} {symbol}'
-                          id='modals.staking.accountsummary.transactionsprogress.stake'
-                          values={{
-                            symbol,
-                            value
-                          }}
-                        />
-                      }
-                      tooltipId='modals.staking.bonding.pending.tooltip'
-                      value={format(new Date(bondingStartDate), 'MMMM d yyyy @ h:mm a')}
-                    />
-                  )
-                )}
+                bondingDeposits.map(({ amount, bondingDays, bondingStartDate, paymentRef }) => (
+                  <Detail
+                    key={paymentRef}
+                    subText={<FormattedMessage defaultMessage='Pending' id='copy.pending' />}
+                    subValue={
+                      <FormattedMessage
+                        defaultMessage='Bonding Period: {bondingDays} {days}'
+                        id='modals.staking.accountsummary.bondingperiod'
+                        values={{
+                          bondingDays,
+                          days:
+                            bondingDays > 1 ? (
+                              <FormattedMessage
+                                defaultMessage='days'
+                                id='modals.staking.warning.content.subtitle.days'
+                              />
+                            ) : (
+                              <FormattedMessage
+                                defaultMessage='day'
+                                id='modals.staking.warning.content.subtitle.day'
+                              />
+                            )
+                        }}
+                      />
+                    }
+                    text={
+                      <FormattedMessage
+                        defaultMessage='Stake {value}'
+                        id='modals.staking.accountsummary.transactionsprogress.stake'
+                        values={{
+                          value: (
+                            <CoinDisplay
+                              coin={coin}
+                              color='gery900'
```

Comment:
Not sure about this, but maybe you meant...
```suggestion
                              color='grey900'
```